components/maps/nigeria-map-demos.tsx

```
@@ -0,0 +1,443 @@
+            path.setAttribute('fill', INACTIVE_COLOR);
+            path.setAttribute('stroke', STROKE_COLOR);
+            path.setAttribute('stroke-width', '0.8');
+            (path as any).style.cursor = 'pointer';
+            (path as any).style.transition = 'fill 0.2s ease';
+
+            path.addEventListener('click', () => {
+              const id = path.getAttribute('id') || '';
+              const name = path.getAttribute('name') || 'Unknown';
+
+              // Reset all states
+              paths.forEach(p => p.setAttribute('fill', INACTIVE_COLOR));
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)